### DISCUSSION TITLE
tractusx-edc 0.5.0-RC5 test results

### DISCUSSION THREAD

**Discussion by stephanbcbauer** [eclipse-tractusx/tractusx-edc#614] [Ideas]
0.5.0-RC5 testing results by @matgnt:

(cross posting my testing results, because it was initially shared only in an SSI realted group)

With this, I think I have finished my basic testing of 0.5.0-RC5. Short summary:

SSI related:

- only a few documentation topics already discussed with @wolf4ood that are easy to integrate
- (I did not do any security related testing yet!) a big open topic I think!!!

NOT SSI, but general migration:

- data mangement API is much more stricter about jsonld, e.g. use meaningful (edc) context information in your request (impact: rather low. needs to be done, but manageable)
- completely different flow / way of how the consumer is able to fetch data (impact on existing toolchains!)
  - EDR no longer consumer EDR around the provider EDR, but pure provider EDRs
  - means, (from my understanding) using a CONSUMER data plane is obsolete (not even possible, at least as it is right now). This might open up questions and how internal integration projects are doing (logging?) data requests right now.
  - also, the properties' field with the _cid_ has been removed from the message to the dynamic receiver endpoint, which might be relevant in case toolchains use this for correlations. (impact: I would say manageable IF _id_ field is stable and reliably the transfer process _id_ as it seems right now.

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
